src/test/java/api/CirculationRulesEngineAPITests.java

```diff
@@ -67,7 +67,7 @@ private Policy applyRequestPolicy(ItemType itemType, String requestType,
       URL url = circulationRulesUrl(
           "/request-policy"
           + "?item_type_id="         + itemType.id
-          + "&request_type_id="         + requestType
+          + "&loan_type_id="         + requestType
```

Comment:
Why have the parameters for a determining a request policy changed?

Reply:
I'm not 100% sure, but the commit associated with the change is named 'Fixed broken tests', so it sounds like there was a bug.

https://github.com/folio-org/mod-circulation/commit/997506afc7aba5b7ecd7ed2bebef3bebf7e59d4a